pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -2431,7 +2452,25 @@ func (c *Cloud) EnsureLoadBalancer(clusterName string, apiService *api.Service,
 			return nil, err
 		}
 	}
-	securityGroupIDs := []string{securityGroupID}
```

Comment:
Might be clearer to keep this line, but the append to it in the EnableSharedSecurityGroupIngress case